xbmc/cores/DataCacheCore.cpp

```diff
@@ -34,20 +34,26 @@ void CDataCacheCore::Reset()
 {
   {
     std::unique_lock<CCriticalSection> lock(m_stateSection);
-
-    m_stateInfo.m_speed = 1.0;
-    m_stateInfo.m_tempo = 1.0;
-    m_stateInfo.m_stateSeeking = false;
-    m_stateInfo.m_renderGuiLayer = false;
-    m_stateInfo.m_renderVideoLayer = false;
+    m_stateInfo = {};
     m_playerStateChanged = false;
   }
-
+  {
+    std::unique_lock<CCriticalSection> lock(m_videoPlayerSection);
+    m_playerVideoInfo = {};
+  }
+  {
+    std::unique_lock<CCriticalSection> lock(m_audioPlayerSection);
+    m_playerAudioInfo = {};
+  }
+  {
+    std::unique_lock<CCriticalSection> lock(m_renderSection);
+    m_renderInfo = {};
+  }
   {
     std::unique_lock<CCriticalSection> lock(m_contentSection);
-
     m_contentInfo.Reset();
   }
+  m_timeInfo = {};
```

Comment:
What about `m_hasAVInfoChanges`? Should that also be reset?

Reply:
done, missed that one